test/jsonapi-response-builder-person.test.js

```diff
@@ -1,21 +1,45 @@
 const test = require('tape');
 const config = require('../config');
 const buildJSONResponse = require('../lib/jsonapi-response.js');
-const JSONAPIResponse = buildJSONResponse(require('./fixtures/example-get-response-person.json'), config);
+var JSONAPIResponse;
 
-test('JSON Response for an object should be structured correctly', function (t) {
+test('Response is built succesfully', (t) => {
+  t.plan(1);
+  t.doesNotThrow(() => {
+    JSONAPIResponse = buildJSONResponse(require('./fixtures/elastic-responses/example-get-response-person.json'), config);
+  }, 'Building response does not throw an error');
+  t.end();
+});
+
+test('JSON Response for an object should be structured correctly', (t) => {
   t.plan(5);
   t.ok(JSONAPIResponse.data, 'Response should contain data field');
   t.ok(JSONAPIResponse.data.attributes, 'Data field should contain attributes field');
-  t.notOk(JSONAPIResponse.data.relationships, 'Data field should not contain relationships field');
+  t.ok(JSONAPIResponse.data.relationships, 'Data field should contain relationships field');
   t.ok(JSONAPIResponse.data.links, 'Data field should contain links field');
-  t.notOk(JSONAPIResponse.included, 'Response should not contain included field');
+  t.ok(JSONAPIResponse.included, 'Response should contain included field');
   t.end();
 });
```

Comment:
Tests that depend on the results of other tests are less portable and maintainable. There's no need to optimise tests, call `buildJSONResponse` in each one.